experimental/LieAlgebras/src/LieAlgebra.jl

```diff
@@ -56,6 +56,13 @@ function basis(L::LieAlgebra, i::Int)
   return L([(j == i ? one(R) : zero(R)) for j in 1:dim(L)])
 end
 
+@doc raw"""
+    characteristic(L::SimpleLieAlgebra{T}) -> Int64
+
+Return the characteristic of the base ring of the Lie algebra `L`.
+"""
+characteristic(L::LieAlgebra{C}) where {C<:RingElement} = characteristic(base_ring(L))
```

Comment:
```suggestion
@doc raw"""
    characteristic(L::LieAlgebra{C}) -> Int64

Return the characteristic of the coefficient ring of the Lie algebra `L`.
"""
characteristic(L::LieAlgebra) = characteristic(coefficient_ring(L))
```
`base_ring` has been removed in https://github.com/oscar-system/Oscar.jl/pull/2528 in favor of `coefficient_ring`.